tests/unit/utils/test_zfs.py

```diff
@@ -891,6 +891,18 @@ class ZfsUtilsTestCase(TestCase):
     This class contains a set of functions that test salt.utils.zfs utils
     '''
     ## NOTE: test parameter parsing
+    def test_is_supported(self):
+        '''
+        Test zfs.is_supported method
+        '''
+        for value in [False, True]:
+            with patch.object(zfs, '_zpool_cmd', MagicMock(return_value=value)):
```

Comment:
We probably don’t need this one as we removed it from the is_supported function.